package/Dockerfile

```diff
@@ -1,25 +1,39 @@
-FROM registry.suse.com/bci/bci-base:15.5
+ARG BCI_VERSION=15.6
 
-ENV SSL_CERT_DIR /etc/rancher/ssl
+FROM registry.suse.com/bci/bci-busybox:${BCI_VERSION} AS final
+FROM registry.suse.com/bci/bci-base:${BCI_VERSION} AS builder
+
+# Creates the base dir for the target image, and hydrates it with the
+# final image's contents.
+RUN mkdir /chroot
+COPY --from=final / /chroot/
 
-RUN zypper -n update && \
-    zypper -n install git-core curl ca-certificates unzip mkisofs xz gzip sed tar openssh-clients && \
+RUN zypper --non-interactive refresh && \
+    zypper --installroot /chroot -n rm busybox-less && \
+    zypper --installroot /chroot -n install \
+        git-core curl mkisofs openssh-clients openssl patterns-base-fips && \
     zypper -n clean -a && \
-    rm -rf /tmp/* /var/tmp/* /usr/share/doc/packages/*
+    rm -rf /chroot/tmp/* /chroot/var/tmp/* /chroot/usr/share/doc/packages/*
 
 RUN useradd -u 1000 machine
+RUN cp /etc/passwd /chroot/etc/passwd
+
+COPY download_driver.sh /chroot/usr/local/bin/
+RUN chmod +x /chroot/usr/local/bin/download_driver.sh
+
+COPY rancher-machine entrypoint.sh /chroot/usr/local/bin/
+RUN chmod 0755 /chroot/usr/local/bin
+
+FROM scratch
```

Comment:
Just adding the same note about the other PR in r/r, to check if the labels could be overwritten or not.

Reply:
I'd say this is less of a problem given the point of `rancher/machine` in its life-cycle. But happy to manually add them if you think that's needed.

Please note that the underlying layers will all be automatically surfaced in the Provenance for projects complying with SLSA.

Reply:
I think that it's always nice to have at least the `org.opencontainers.image.*`, so we know where it's coming from (good for our automation). I check and it seems to be done in the GHA release pipeline - https://github.com/rancher/machine/blob/master/.github/workflows/release.yaml#L71-L77.